### DISCUSSION TITLE
Poor quality after training VITS on M-AILABS (Russian)

### DISCUSSION THREAD

**Discussion by andreibezborodov** [coqui-ai/TTS#1472] [General Q&A]
Hello!

I am constantly having bad results after training VITS on M-AILABS for Russian speakers.  The resulting voice "shuffles" the letters in the words for all speakers. I have already made several attempts, tried to use phonemes, but the result is the same.

Attaching examples of voice and TB with config.

https://tensorboard.dev/experiment/YQb8cw9PRqOB6MgxpAinMA/#scalars

https://user-images.githubusercontent.com/23527206/161498367-f81bfb3f-8017-457c-96f4-c06dda18b3b0.mp4

https://user-images.githubusercontent.com/23527206/161498372-794485e8-43d2-4b3a-ba3b-87d6b0303faa.mp4

Thank you in advance!

**Comment by Theodotus1243** [comment]
@andreibezborodov 
Try turn off phonemizer 
The voice and training curves is ok, but phonemizer can disrupt pronunciation

Pass into model Graphemes object of ru alphabet
Also set use_sdp : True, it allways help
